monitor/utils/influxdb/connector.py

```diff
@@ -66,6 +65,27 @@ def get_cost_measurements(self):
 
         return out
 
+    def get_stream_measurements(self):
+
+        out = {}
+
+        for i in self.get_real_output_flux():
+            out[i['time']].update({'real_output_flux': i['value']})
+
+        for i in self.get_expected_output_flux():
```

Comment:
Isn't expected output flux equal to input flux?

Reply:
No, the expected_output here is the value calculated with replicas/expected_time, in a perfect scenario they do are the same, but since the input is looked at run time, not necessarily it will even be an integer, making the expected output different.